apiserver/facades/client/machinemanager/machinemanager_test.go

```diff
@@ -601,6 +602,19 @@ func (s *MachineManagerSuite) TestUpgradeSeriesComplete(c *gc.C) {
 	c.Assert(err, jc.ErrorIsNil)
 }
 
+func (s *MachineManagerSuite) TestApplications(c *gc.C) {
+	s.setupUpdateMachineSeries(c)
+	apiV5 := machinemanager.MachineManagerAPIV5{MachineManagerAPI: s.api}
+	args := params.Entities{
+		Entities: []params.Entity{{
+			Tag: names.NewMachineTag("0").String(),
+		}},
+	}
+	results, err := apiV5.Applications(args)
+	c.Assert(err, jc.ErrorIsNil)
+	c.Assert(results.Results[0].Result, jc.SameContents, []string{"foo"})
```

Comment:
Where does "foo" come from? It seems that this test just knows that "foo" is the right answer but a reader of the test may be a bit puzzled at first as to where that result came from when trying to quickly diagnose a test failure.  Perhaps `[]string{s.Machine.Unit[0],ApplicationsName}` or wherever "foo" might live.